components/shared/SubmitButton.tsx

```diff
@@ -1,52 +1,47 @@
 import { ButtonHTMLAttributes, FC, useMemo } from "react";
 import { css } from "@emotion/react";
 import { Theme } from "@mui/material";
-import { useCENNZWallet } from "@/providers/CENNZWalletProvider";
 import CENNZIconSVG from "@/assets/vectors/cennznet-icon.svg";
 import MetaMaskSVG from "@/assets/vectors/metamask.svg";
 import { useMetaMaskWallet } from "@/providers/MetaMaskWalletProvider";
+import { useWalletProvider } from "@/providers/WalletProvider";
 
 interface SubmitButtonProps {
-	requireCENNZnet: boolean;
-	requireMetaMask: boolean;
+	requireMetaMask?: boolean;
```

Comment:
since `SubmitButton` is fully aware of `selectedWallet` inside, so it can determine whether it needs MetaMask or not. The only exception is `BridgeForm` when `selectedWallet` is `"CENNZnet"`, in that case, we want to have a prop called `forceRequireMetaMask` here instead